src/maps/deployment.ts

```diff
@@ -47,9 +47,9 @@ export const mapDeploymentUpdateToDeploymentUpdateRequest: MapFunction<Deploymen
   return mapped
 }
 
-export const mapDeploymentFlowRunCreateToDeploymentFlowRunRequest: MapFunction<DeploymentFlowRunCreate, DeploymentFlowRunRequest> = function(source: DeploymentFlowRunCreate): DeploymentFlowRunRequest {
-  const { parameters, state, schema, ...rest } = source
-  const mapped = mapCamelToSnakeCase<DeploymentFlowRunRequest>(rest)
+export const mapDeploymentFlowRunCreateToDeploymentFlowRunRequest: MapFunction<DeploymentFlowRunCreate, DeploymentFlowRunRequest> = function(source) {
+  const { parameters, state, schema, empiricalPolicy, ...rest } = source
+  const mapped = mapCamelToSnakeCase(rest) as DeploymentFlowRunRequest
```

Comment:
You mentioned wanting to be more explicit in mappers. Feel free to remove this helper in favor of an explicit object map.

Reply:
I will do a follow-up PR that removes `mapCamelToSnakeCase`, there are only 6 profiles that use it

Reply:
https://github.com/PrefectHQ/orion-design/pull/1087